docs/docs/Integrations/Google/integrations-setup-google-cloud-vertex-ai-langflow.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Integrate Google Cloud Vertex AI with Langflow'
+slug: /integrations-setup-google-cloud-vertex-ai-langflow
+sidebar_position: 2
+description: "A comprehensive guide on creating a Google OAuth app, obtaining tokens, and integrating them with Langflow's Google components."
+---
+
+Langflow integrates with the [Google Vertex AI API](https://console.cloud.google.com/marketplace/product/google/aiplatform.googleapis.com) for authenticating the [Vertex AI embeddings model](/components-embedding-models#vertexai-embeddings) and [Vertex AI](/components-models#vertexai) components.
+
+Learn how to create a service account JSON in Google Cloud to authenticate Langflow’s Vertex AI components.
+
+## Create a project and enable the Vertex AI API
+
+1. Navigate to the [Google Cloud Console](https://console.cloud.google.com/).
+2. Click **Select a project**, and then click **New Project** to create a new project.
```

Comment:
I think this can be simplified to "Create a new project". Also suggest adding a link to the Google Cloud Console docs that describe how to create a project.